<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScenes</class>
 <widget class="QWidget" name="GameScenes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="Line" name="line_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Credits:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Ending Player #2:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="ending_credits_combo"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Ending Player #3:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="Line" name="line_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="company_combo">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="ending_player1_combo"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Copyright/Company:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ending Player #1:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Intro (Unbeaten Game):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="intro_combo">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Intro (beaten Game)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>[optional]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="intro_beaten_combo"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>Ending Player #4:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="ending_player4_combo"/>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="ending_player2_combo"/>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="ending_player3_combo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
